lib/plugins/executable/index.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const fs = require('fs');
+const os = require('os');
+const streamPromise = require('stream-promise');
+const fse = BbPromise.promisifyAll(require('fs-extra'));
+const isStandaloneExecutable =
+  require('../../utils/isStandaloneExecutable') && process.platform !== 'win32';
```

Comment:
Why do we exclude `win32` here?

Reply:
Because on Windows we will publish to [chocolatey](https://chocolatey.org/) package manager, therefore users will not use `serverless upgrade` but `choco upgrade` (both `sls upgrade` and `sls update` will not be available there)

Reply:
I see. Would be interesting to see how this shakes out once we add Homebrew / Macports support. In that case would it mean that we drop `darwin` support for the binary install script?

Same question might also apply for Linux, although there are many more package managers we'd have to support (and given that a binary installation via `curl` seems more reasonable).

Reply:
> to see how this shakes out once we add Homebrew / Macports support

I would cover that in next step (if there's a demand), otherwise we'll never have it ready :)

Then we can probably come up with some additional detection, to exclude those cases as well